platform/platform_tests/distance_tests.cpp

```diff
@@ -299,7 +301,7 @@ UNIT_TEST(Distance_FormattedDistance)
     {Distance(10000,   Units::Meters),     10,   Units::Kilometers, "10",   "10 km"},
     {Distance(10499.9, Units::Meters),     10,   Units::Kilometers, "10",   "10 km"},
     {Distance(10501,   Units::Meters),     11,   Units::Kilometers, "11",   "11 km"},
-    {Distance(287'386, Units::Meters),     290,  Units::Kilometers, "290",  "290 km"},
+    {Distance(287'386, Units::Meters),     287,  Units::Kilometers, "287",  "287 km"},
 
     // From Feet to Miles
```

Comment:
Are tests "From Feet to Feet" missing?

Reply:
Lines 267-274

Reply:
Aren't these lines about kilometers to kilometers?

Reply:
These lines were about `Feet to Feet` at first stage of the PR 😀 
Now check lines 275-282